types/sequelize/index.d.ts

```diff
@@ -3759,22 +3768,22 @@ declare namespace sequelize {
          *
          * @see    {Sequelize#query}
          */
-        findAll(options?: FindOptions): Promise<TInstance[]>;
```

Comment:
Could you add these lines back without causing issues? I think this would allow for both generic and non-generic usages of the function and reduce the number of potential breaking changes. For example I'm having an issue where I'm calling this method via a type alias.